README.md

```diff
@@ -0,0 +1,194 @@
+### What's Open ID Connect ?
+
+Open ID is a protocol that allows multiple clients to delegate the Authentication to one or multiple Authority servers.
+Facebook, Google, Twitter, are all well-known as OpenId Authority Servers.
+
+### Why should I want/need my own Open ID Connect Server?
+
+There are many scenarios on when this may be useful, 
+
+ 1. Having multiple applications with one central place for authentication.
+ 2. Having non-trusted clients (like JavaScript, Desktop or Mobile applications).
+ 3. Needs or wants to use single sign-on for multiple services.
+
+In general terms, you want to have a centralized authentication server/login and then any allowed clients
+(yours or not) can use the authentication server to handle identity and claims.
+
+--------------
+
+## Getting Started
+
+To use OpenIddict Server you need to include OpenIddict as a dependency in your project.json:
+```
+    "dependencies": {
+        "OpenIddict": "1.0.0-*"
+    },
+```
+
+In ConfigureServices there's a handy extension method for IdentityBuilder that you need to set up.
+Here is a complete ConfigureServices including Identity, Mvc and EntityFramework:
+
+```csharp
+public void ConfigureServices(IServiceCollection services) {
+    services.AddMvc();
+
+    services.AddEntityFramework()
+        .AddSqlServer()
+        .AddDbContext<ApplicationDbContext>(options =>
+            options.UseSqlServer(Configuration["Data:DefaultConnection:ConnectionString"]));
+
+    services.AddIdentity<ApplicationUser, IdentityRole>()
+        .AddEntityFrameworkStores<ApplicationDbContext>()
+        .AddDefaultTokenProviders()
+        .AddOpenIddict();
+
+    /// .... other services you may have
+}
+
+in your Configure method, you will need to use it:
+
+```csharp
+public void Configure(IApplicationBuilder app) {
+    app.UseIdentity();
+    
+    // any external provider like, app.UseGoogleAuthentication, app.UseFacebookAuthentication, etc..
+    
+    app.UseOpenIddict(options => {
+        // options
+    });
+}
+```
+
+> `UseOpenIddict()` must be **AFTER** `app.UseIdentity()` and any external providers.
+
+### Configuration & Options.
+
+<p>OpenIddict have multiple options you can set to customize for your requirements/needs.<p>
+
+<table>
+<thead>
+    <tr><th>Options</th><th>type</th><th>Description</th><th>Default</th></tr>
+</thead>
+<tbody>
+<tr>
+<td>AuthenticationScheme</td>
+<td>string</td>
+<td>
+    The base address used to uniquely identify the authorization server.<br>
+    The URI must be absolute and may contain a path, but no query string or fragment part.<br>
+    Unless AllowInsecureHttp has been set to true, an HTTPS address must be provided.
+</td>
+<td><code>oidc-server</code></td>
+</tr>
+<tr>
+  <td>Issuer</td>
+  <td>Uri</td>
+  <td>The base address (absolute) used to uniquely identify the authorization server.<br>
+      <sub><i>Unless <code>AllowInsecureHttp</code> has been set to <code>true</code>, an HTTPS address must be provided.</i></sub>
```

Comment:
I should probably fix ASOS' documentation to mention this property is now optional. When left to null, the issuer property is automatically inferred from the request URL.